docs/dapp/opl/setup.md

```diff
@@ -30,13 +30,13 @@ Let's create a new Hardhat project.
 
 :::info
 
-Currently we are compatible with Hardhat up to `2.12.7`. You may need to
+Currently we are compatible with Hardhat up to `2.16.0`. You may need to
```

Comment:
Since we're using the tilde, we don't care about the patch versions right?

```suggestion
Currently we are compatible with Hardhat up to `2.16`. You may need to
```

Reply:
Fair 👍